cms/jinja2/templates/pages/statistical_article_page.html

```diff
@@ -174,7 +174,7 @@ <h3 class="ons-corrections-notices__banner-title ons-u-mb-2xs">{{ corrections_an
             <h2 class="headline-figures__heading ons-u-fs-m">{{ _("Headline facts and figures") }}</h2>
             <div class="ons-grid ons-grid-flex-gap ons-grid-flex-gap--24 headline-figures__grid">
                 {% for figure in page.headline_figures %}
-                    {% include_block figure %}
+                    <div class="ons-grid__col">{% include_block figure %}</div>
```

Comment:
```suggestion
                    <div class="ons-grid__col ons-grid__col--flex-col ons-grid__col--stretch">{% include_block figure %}</div>
```

To keep the cards the same height even if the content is different